gradle/build-logic/convention/src/main/java/app/tivi/gradle/Spotless.kt

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023, Christopher Banes and the Tivi project contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package app.tivi.gradle
+
+import com.diffplug.gradle.spotless.SpotlessExtension
+import org.gradle.api.Project
+import org.gradle.api.artifacts.VersionCatalogsExtension
+import org.gradle.kotlin.dsl.configure
+import org.gradle.kotlin.dsl.getByType
+
+fun Project.configureSpotless() {
+    val libs = extensions.getByType<VersionCatalogsExtension>().named("libs")
+    val ktlintVersion = libs.findVersion("ktlint").get().requiredVersion
```

Comment:
To avoid this we could also remove our version override and use what comes as default with Spotless